conda_build/windows.py

```diff
@@ -121,7 +121,6 @@ def build(m):
             os.makedirs(path)
 
     src_dir = source.get_dir()
-    bld_bat = join(m.path, 'bld.bat')
```

Comment:
Sorry, the monitors they have us using here are truly miserable contrast.  Thanks for pointing that out. =)